go/libkb/skb.go

```diff
@@ -24,6 +25,18 @@ import (
 	triplesec "github.com/keybase/go-triplesec"
 )
 
+func DumpKey(g *GlobalContext, name string, b []byte) {
+	tmp, err := ioutil.TempFile(os.TempDir(), "dump-"+name)
+	if err != nil {
+		g.Log.Warning("Failed to dumpKey %s: %s", name, err)
+		return
+	}
+	g.Log.Notice("DUMPKEY %s -> %s", name, tmp.Name())
+	buf := bytes.NewBuffer(b)
+	io.Copy(tmp, buf)
+	tmp.Close()
+}
```

Comment:
If you want to keep it in, maybe call it something with Debug in the name, or leave a comment about it?